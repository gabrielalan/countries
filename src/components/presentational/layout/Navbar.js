import React from 'react';

import './Navbar.scss';

export default function Navbar({ children }) {
  return <div className="navbar">
    <h1 className="navbar__logo">FancyCountries</h1>
    <div className="navbar__right-side">
      {children}
    </div>
  </div>;
}