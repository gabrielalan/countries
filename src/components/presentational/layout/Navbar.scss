@import '../../../styles/base';

.navbar {
  background-color: color('navbar', 'background');
  display: flex;
  align-items: center;
  justify-content: space-between;
  padding: gap(2) gap(2) gap(30);
  margin-bottom: -(gap(15));

  &__logo {
    margin: 0;
    color: white;
  }

  &__right-section {
    display: flex;
    align-items: center;
    color: white;
  }
}