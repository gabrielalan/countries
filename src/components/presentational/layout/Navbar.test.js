import React from 'react';
import renderer from 'react-test-renderer';
import Navbar from './Navbar';

describe('Navbar', () => {
  it('should render text normally', () => {
    const tree = renderer.create(<Navbar>inside the right section</Navbar>).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
