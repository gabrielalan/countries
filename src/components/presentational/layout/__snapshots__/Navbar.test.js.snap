// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navbar should render text normally 1`] = `
<header
  className="navbar"
>
  <h1
    className="navbar__logo"
  >
    FancyCountries
  </h1>
  <section
    className="navbar__right-section"
  >
    inside the right section
  </section>
</header>
`;
